Gordon360/Services/AddressesService.cs

```diff
@@ -15,15 +16,8 @@ public AddressesService(CCTContext context)
             _context = context;
         }
 
-        public IEnumerable<States> GetAllStates()
-        {
-
-            return _context.States.AsEnumerable();
-        }
+        public IEnumerable<States> GetAllStates() => _context.States.AsEnumerable();
```

Comment:
Why does this need to be an explicit enumerable? Is it not fine to just return the dbset?

Reply:
I think it's fine to just return the DbSet and let it be implicitly converted. It will be enumerated when serializing to JSON either way.